packages/browser/src/Widget/Server/Workspace/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Workspace Module
+
+The Workspace Module is an extension of the SPARQL Module that adds functionality to query and visualize Workspace, and CityGML Versioning data stored in an RDF Triple-store from a UD-Viz interface.
+
+See the [SPARQL module](../SPARQL/) for more documentation on generic SPARQL functions.
+
+## Workspace functionalities
+This module extends the functionality of the [SPARQL module](../SPARQL/) with the ability to update the displayed 3D model by clicking on **Version** and **VersionTransition**. These nodes are displayed in a D3 graph created from Workspace and Versioning graph data from a SPARQL backend service.
+
+If a **VersionTransition** is selected and highlight changes between the related versions of the 3D city model will be highlighted.
+
+## Data requirements
+Note that for this functionality to work RDF data in the SPARQL endpoint must be conformant to the ontologies proposed here:
+- [CityGML 3.0](https://dataset-dl.liris.cnrs.fr/rdf-owl-urban-data-ontologies/Ontologies/CityGML/3.0/)
+- [CityGML 3.0 Workspace ADE](https://dataset-dl.liris.cnrs.fr/rdf-owl-urban-data-ontologies/Ontologies/Workspace/3.0/)
+
+Additionally, the 3DTiles used to visualize the 3D model(s) must conform to the 3DTiles Temporal batch table hierarchy extension supported by the [Temporal](../../Temporal/Temporal.js) widget.
+
+## Setup
+To configure the SPARQL workspace module extension
+- a 3DTiles tileset with the temporal module/widget
```

Comment:
no need of the module (except the creaton function but this function will be deprecated since we want to fit itowns api) Temporal only the tile are required